packages/features/portfolio/src/components/deprecation-message/deprecation-message.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, styled, useTheme } from '@mui/material';
+import { Body, Button, LargeInputTextEmphasized } from '@notional-finance/mui';
+import { FormattedMessage } from 'react-intl';
+
+export const DeprecationMessage = () => {
+  const theme = useTheme();
+
+  return (
+    <Container>
+      <Box sx={{ marginBottom: theme.spacing(6) }}>
+        <LargeInputTextEmphasized sx={{ marginBottom: theme.spacing(1) }}>
+          <FormattedMessage
+            defaultMessage={
+              'Notional V2 is being deprecated. Notional V3 is Live!'
+            }
+          />
+        </LargeInputTextEmphasized>
+        <Body sx={{ fontSize: theme.spacing(2) }}>
+          <FormattedMessage
+            defaultMessage={
+              '<a>Important:</a> Notional V2 will be fully deprecated by the end of July. Read the updated deprecation plan:'
+            }
+            values={{
+              a: (msg: string) => (
+                <Box component={'span'} sx={{ fontWeight: 600 }}>
+                  {msg}
+                </Box>
+              ),
+            }}
+          />
+        </Body>
+      </Box>
+      <ButtonBox>
+        <Button
+          variant="outlined"
+          size="large"
+          href="https://blog.notional.finance/notional-v2-deprecation-plan/"
```

Comment:
Ok so we just need to replace this URL right?

Reply:
Yes, we need to replace that with the new one whenever Teddy publishes it